src/clean_data.py

```diff
@@ -0,0 +1,100 @@
+import pandas as pd
+from glob import glob
+from src.helpers import filter_on_year
+import numpy as np
+
+def read_data(PATH,df_names):
+    """
+    Reads csv and excel files
+    :param str PATH: file path pointing to the directory with the data.
+    :return `dict`:a hashmap of pandas dataframes with value as df
+    """
+
```

Comment:
as per PEP8 req'ts docstrings, there are some identation rules to be followed when writing docstrings. also, you forgot to describe `df_names`